src/VirtualClient/VirtualClient.Actions/DiskPerformanceWorkloadExecutor.cs

```diff
@@ -397,17 +397,16 @@ protected virtual IEnumerable<DiskPerformanceWorkloadProcess> CreateWorkloadProc
             return this.Logger.LogMessage($"{this.GetType().Name}.CreateProcesses", telemetryContext, () =>
             {
                 List<DiskPerformanceWorkloadProcess> processes = new List<DiskPerformanceWorkloadProcess>();
-                string testedInstance = this.DiskFilter;
 
                 if (string.Equals(processModel, WorkloadProcessModel.SingleProcess, StringComparison.OrdinalIgnoreCase))
                 {
-                    processes.Add(this.CreateWorkloadProcess(executable, commandArguments, testedInstance, disks.Select(disk => disk).ToArray()));
+                    processes.Add(this.CreateWorkloadProcess(executable, commandArguments, string.Join(",", disks.Select(d => d.DevicePath)), disks.ToArray()));
```

Comment:
The testedInstance was DiskFilter, if we do diskpath instead, we might need to sort it because the disk could come as different order, and then when we run the query it will come out different for different VM.

Reply:
I will change it back to the disk filter for now. My worry is that in doing so, we will not be able to distinguish variations of FIO in the future where there are different numbers of disks used. The results for running SingleProcess on 2 vs. 16 disks may not be equal but they will look equal in the data if we use the disk filter because the value will be the same.

Reply:
Per our conversation, I am using the {ProcessModel},{DiskFilter},{DiskCount} for the metric categorization/tested instance (e.g. SingleProcess,BiggestSize,16 or SingleProcessPerDisk,BiggestSize,16).